src/pymordemos/coercivity_estimation_scm.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+"""Simplified version of the thermalblock demo to showcase the successive constraints method."""
```

Comment:
That should be a docstring for `main` so that it can become part of the typer-generated documentation.